core/src/main/java/io/aiven/klaw/service/DefaultDataService.java

```diff
@@ -302,6 +306,14 @@ public List<KwProperties> createDefaultProperties(int tenantId, String mailId) {
             KwConstants.MAIL_TOPICUPDATEREQUEST_CONTENT,
             "Email notification body for a new Topic Update Request");
     kwPropertiesList.add(kwProperties37);
+
+    KwProperties kwProperties38 =
+        new KwProperties(
+            KLAW_EXTRA_PERMISSION_TOPIC_PROMOTION_KEY,
+            tenantId,
+            "false",
+            "Need of an extra permission APPROVE_TOPICS_PROMOTION to promote topics");
```

Comment:
"Optional permission APPROVE_TOPICS_PROMOTION to allow users to promote topics but not create topics"